### PR TITLE
Improved build instructions

### PR DESCRIPTION
**Pull request by Wikinaut**
Added lines from http://askubuntu.com/a/27679

I had the same problems. The proposed change works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,11 @@ Building instructions
 `unpaper` uses GNU Autotools for its build system, so you should be
 able to execute the same commands used for other software packages:
 
+    libtoolize --force
+    aclocal
+    autoheader
+    automake --force-missing --add-missing
+    autoconf
```
Comment (Flameeyes) [inline README.md]: This should actually be `autoreconf -i` (and only when building out of the Git repository).

**Comment by amitdo** [comment]
You might want to add `autogen.sh` script file instead, and tell the users to run it before `configure`.

**Comment by Wikinaut** [comment]
@amitdo There was no `autogen.sh` in your source code bundle. This was the reason why I was puzzled how to build the current version.

**Comment by Wikinaut** [comment]
On example is here: https://github.com/tesseract-ocr/tesseract/blob/master/autogen.sh (I use unpaper as preprocessor for tesseract).

**Comment by Flameeyes** [comment]
I explicitly don't want to have an `autogen.sh` file because it's unnecessary. And unpaper does not use libtool, so that part is also unnecessary.

A single `autoreconf -i` will do the right thing.

**Comment by Wikinaut** [comment]
@Flameeyes then, perhaps, this `autoreconf -i` should be added to the build instructions ?

**Pull request closed without merging**